knowledge/cars/delorean/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: arxspecialist
+domain: automobile
+seed_examples: What engines were considered in the making of the DMC Delorean?
+- answer: | The Ford Cologne V6 engine was first considered. The 2.0 engine from a the Citroen CX2000 was then considerend and eventually dropped in favor of the fuel injected v6 PRV (Peugeot-Renault-Volvo) engine.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!